java/cdk/src/test/java/sleeper/cdk/UtilsPropertiesTest.java

```diff
@@ -155,34 +150,10 @@ void shouldFailWhenTableNameCannotBePartOfAValidBucketName() throws IOException
 
             // When / Then
             InstanceProperties load = new InstanceProperties();
-            Function<String, String> context = cdkContextWithPropertiesFile();
+            Function<String, String> context = cdkUpgradeContextWithPropertiesFile(tempDir);
             assertThatThrownBy(() -> Utils.loadInstanceProperties(load, context))
                     .isInstanceOf(IllegalArgumentException.class)
                     .hasMessage("Sleeper table bucket name is illegal: sleeper-valid-id-table-example--invalid-name-tab$$-le");
         }
     }
-
-    private Function<String, String> cdkContextWithPropertiesFile() {
-        return Map.of("propertiesfile", tempDir.resolve("instance.properties").toString())::get;
-    }
```

Comment:
Please can we have this helper method back?

Reply:
We need to avoid changing the test data supplied to these tests. If they fail with it as it was before, it's a problem with the implementation.